ocaml_version.ml

```diff
@@ -311,6 +312,7 @@ module Configure_options = struct
     | `Force_safe_string -> "force safe string mode"
     | `Frame_pointer -> "frame pointer"
     | `No_naked_pointers -> "forbid unboxed pointers"
+    | `No_naked_pointers_checker -> "forbid unboxed pointers and print a warning"
```

Comment:
```suggestion
    | `No_naked_pointers_checker -> "enable the naked pointers checker"
```
(naked pointers are still allowed - it actually conflicts `` `No_naked_pointers``)